src/Service/EloquaApiClient.php

```diff
@@ -152,7 +156,7 @@ public function getAccessTokenByRefreshToken() {
    */
   private function doTokenRequest(array $params, $res = 'token') {
     // Guzzle Client.
-    $guzzleClient = new GuzzleClient([
+    $guzzleClient = $this->httpClientFactory->fromOptions([
```

Comment:
surely this shouldnt be `$guzzleClient` as per the line 46 where its `protected $httpClientFactory;`